### DISCUSSION TITLE
Help! Accepted Assignments are Empty

### DISCUSSION THREAD

**Discussion by MisterBrash** [community/community#116922] [GitHub Education]
### Hi everyone,

When students accept assignments the forked assignment is _**empty**_, not even a README.md
Just started happening this morning - has been working fine up until now.

I've deleted the assignment and recreated, used a different repository... same outcome.

Assignments created _yesterday_ seem to work just fine.

**Comment by samus-aran** [comment]
Hi @MisterBrash 

Are you still experiencing this?

**Reply by MisterBrash** [reply]
Hi, thanks for a follow-up. The problem seemed to resolve itself the next day. But it seems nobody else had that same problem that day... I have zero clue what it could have been.
